### PR TITLE
MBS-8088 fix setup docs, add {{%plugins-setup%}} shortcode

### PR DESCRIPTION
**Pull request by dsvoronin**

### CODE REVIEW

File: docs/themes/book/layouts/shortcodes/project-setup.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        eachPlugin {
+            String pluginId = requested.id.id
+            if (pluginId.startsWith("com.avito.android")) {
+                def artifact = pluginId.replace("com.avito.android.", "")
+                useModule("com.avito.android:$artifact:$avitoToolsVersion")
+            }
+        }
+    }
+}
+```
```
Comment (eugene-krivobokov) [inline docs/themes/book/layouts/shortcodes/project-setup.html]: Let's move this file to `docs/layouts/shortcodes` if it's not a part of the book theme.

File: docs/content/docs/projects/BuildChecks.md [hunk 2]
```diff
@@ -17,24 +17,7 @@ plugins {
...
-            String pluginId = requested.id.id
-            if (pluginId.startsWith("com.avito.android")) {
-                def artifact = pluginId.replace("com.avito.android.", "")
-                useModule("com.avito.android:$artifact:$avitoToolsVersion")
-            }
-        }
-    }
-}
-```
+{{%project-setup%}}
```
Comment (eugene-krivobokov) [inline docs/content/docs/projects/BuildChecks.md]: `project` is a common term. What about more specific name? For instance, `plugins-setup`.

File: docs/themes/book/layouts/shortcodes/project-setup.html (hunk 1 above)
Comment (sergioboi) [reply docs/themes/book/layouts/shortcodes/project-setup.html]: What is it? Is the shortcode some concept of our theme?

Comment (eugene-krivobokov) [reply docs/themes/book/layouts/shortcodes/project-setup.html]: Shortcode is a Hugo feature. Theme mostly mirrors a structure. Thus it brings its own resources and can override others.

https://gohugo.io/templates/shortcode-templates/#shortcode-template-lookup-order

Comment (dsvoronin) [reply docs/themes/book/layouts/shortcodes/project-setup.html]: moved

File: docs/content/docs/projects/BuildChecks.md (hunk 2 above)
Comment (dsvoronin) [reply docs/content/docs/projects/BuildChecks.md]: renamed

File: docs/layouts/shortcodes/plugins-setup.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+            name = "Avito bintray"
+            url = "https://dl.bintray.com/avito/maven"
+        }
+    }
+    resolutionStrategy {
+        eachPlugin {
+            String pluginId = requested.id.id
+            if (pluginId.startsWith("com.avito.android")) {
+                def artifact = pluginId.replace("com.avito.android.", "")
+                useModule("com.avito.android:$artifact:$avitoToolsVersion")
```
Comment (eugene-krivobokov) [inline docs/layouts/shortcodes/plugins-setup.html:16]: It'd good to show an example where to put this property.

**Review by eugene-krivobokov — APPROVED** [review]

File: docs/layouts/shortcodes/plugins-setup.html (hunk 3 above)
Comment (dsvoronin) [reply docs/layouts/shortcodes/plugins-setup.html:16]: done

**Pull request merged**